app/models/campaign.rb

```diff
@@ -55,6 +55,46 @@ def last_contribution
     ).order(created_at: :desc).first&.created_at
   end
 
+  def seller_distributor_pairs
```

Comment:
nit: mind writing a test? You can defer til we decide on the API though

Reply:
yeah sure happy to write a test. i might merge this first but can we pair on it? i've been trying to write one but i have no idea where any of the variables are coming from